app/jobs/csv_export_job.rb

```diff
@@ -31,9 +24,11 @@ def export_task(csv_export)
     csv_export.status! :started
     deploy_csv_export(csv_export)
     notify_of_creation(csv_export)
-  rescue
+  rescue Exception => e
```

Comment:
rescuing `Exception` is generally not a good idea. Take a look at http://ruby-doc.org/core-2.2.0/Exception.html to see all the things this might be swallowing - among other things it eats OOM errors and signals